views/region/index.erb

```diff
@@ -0,0 +1,31 @@
+<% @page_title = "Custom Regions" %>
+
+<%== part(
+  "components/page_header",
+  right_items: ((@regions&.count > 0) && has_project_permission("Region:create")) ? [
```

Comment:
```suggestion
  right_items: ((!@regions.empty?) && has_project_permission("Region:create")) ? [
```

I don't think this page is rendered without `@regions` being set.  Coverage testing should tell you for sure.  Also, it's better to use `!empty?` instead of `count > 0`.